contracts/Escrow.sol

```diff
@@ -171,14 +192,14 @@ contract Escrow {
             status == EscrowStatuses.Partial,
             "Escrow not in Pending or Partial status state"
         );
-        intermediateResultsUrl = _url;
-        intermediateResultsHash = _hash;
-        emit IntermediateStorage(_url, _hash);
+        recordingOracleResultsUrl = _url;
+        recordingOracleResultsHash = _hash;
+        emit RecordingOracleResults(_url, _hash);
     }
 
     function bulkPayOut(
```

Comment:
While you're at it, let's rename these to `bulkPayment`. This won't then be confused with the PayPal payouts.